README.md

```diff
@@ -84,18 +84,19 @@ Of course it sometimes includes google's official one.
 
 ## TaaS
 TaaS stands for Test as a Service. Usually, TaaS provides us test platform like running test cases and its report.
+Some services are interesting since they don't provide only **running tests on their device farm**
 
 - [Cloud Test Lab](https://developers.google.com/cloud-test-lab/)
 - [AWS Device Farm](https://aws.amazon.com/jp/device-farm/)
-    - [AppThwack](https://appthwack.com/landing)
 - [SauceLabs](https://saucelabs.com/)
-    - [TestObject](https://testobject.com/)
 - [Testdroid](http://testdroid.com/)
 - [Scirocco Cloud](http://www.scirocco-cloud.com/ja/price.html)
 - [Remote TestKit](https://appkitbox.com/)
-- [Xamarin Test Cloud](http://xamarin.com/test-cloud)
+- [App Center](https://appcenter.ms)
 - [Kobiton](https://kobiton.com)
 - [Mobilebox](http://mobileboxlab.com/en/)
+- [HeadSpin](https://www.headspin.io)
+- [test.ai](http://test.ai)
```

Comment:
HeadSpin is kind of measuring performance service on real devices which exists in various countries. You can figure out performance issue **before** releasing.
test.ai is mobile-focused AI-driven test conducting service.